driver-portal/src/ClientApp/src/app/submission-requirements/submission-requirements.component.html

```diff
@@ -264,3 +268,4 @@ <h2>Definitions</h2>
     </div>
```

Comment:
line 208 below, the definitions, does not exist on the partner portal submission requirements. I didn't see the submission requirements in the wireframes, should they be on both?

Reply:
The definitions are in the case search component. I will move them to the submission requirements page so that it will be same on both the portals.